flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -2758,4 +2758,22 @@ void OmpAttributeVisitor::AddOmpRequiresToScope(Scope &scope,
   } while (!scopeIter->IsGlobal());
 }
 
+void OmpAttributeVisitor::IssueNonConformanceWarning(
+    llvm::omp::Directive D, parser::CharBlock source) {
+  std::string warnStr = "";
+  std::string dirName = llvm::omp::getOpenMPDirectiveName(D).str();
+  switch (D) {
+  case llvm::omp::OMPD_master:
+    warnStr = "OpenMP directive " + dirName +
+        " has been deprecated, please use masked instead.";
+    break;
+  case llvm::omp::OMPD_target_loop:
+  default:
+    warnStr = "Usage of directive " + dirName +
+        "is non-confirming to OpenMP standard.";
```

Comment:
```suggestion
    warnStr = "OpenMP directive '" + dirName +
        "' has been deprecated.";
```

Reply:
Please only commit this suggestion, if the warning is only hit for deprecated OpenMP directives.

Nit: In any case there's a missing space after the directive name and I would suggest to enclose the directive names in single quotes.

Reply:
Sure. Thank you!